software/matomo.yml

```
@@ -1,6 +1,6 @@
 name: Matomo
 website_url: https://matomo.org/
-description: Google Analytics alternative that protects your data and your customers' privacy.
+description: Analytics with protection for your data and your customers' privacy (alternative to Google Analytics).
```

```suggestion
description: Web analytics that protects your data and your customers' privacy (alternative to Google Analytics).
```

Better description (?)